packages/webapp/src/components/AddFarm/FarmAddress/index.js

```diff
@@ -114,7 +114,7 @@ class FarmAddress extends Component {
           required: (val) => val && val.length,
           length: (val) => val && val.length > 2
         }} onBlur={this.handleBlur()}/>
-        <div>Street address or comma separated latitude and longitude (e.g. 49.250945, -123.238492)</div>
+        <div style={{ width: '250px' }}>Street address or comma separated latitude and longitude (e.g. 49.250945, -123.238492)</div>
```

Comment:
Please, move styling to this files stylesheet. Also, the text is looking left aligned. can you make it so its center aligned ?